README.md

```diff
@@ -377,270 +366,35 @@ signIn.off('ready');
 signIn.off('ready', onReady);
 ```
 
-### getTransaction
+### authClient
 
-Returns [authentication transaction information](https://developer.okta.com/docs/api/resources/authn#transaction-model) given a `stateToken`.
+Returns the underlying `@okta/okta-auth-js` object used by the widget
 
-```javascript
-signIn.getTransaction(stateToken)
-  .then(function (transaction) {
-    console.log(transaction.status);
-    console.log(transaction.user.profile.login);
-    // PASSWORD_EXPIRED
-    // user@example.com
-  })
-  .catch(function (err) {
-    console.log(error.message);
-  });
-```
-
-When the `getTransaction` method resolves, it returns a **transaction** object that encapulates the [state authentication flow](https://developer.okta.com/docs/api/resources/authn#transaction-model). The **transaction** contains metadata about the current state, and provides methods that can be used to progress into another state.
-
-> For a list of examples, see the [`transaction`](https://github.com/okta/okta-auth-js/#locked_out) documentation.
-
-### session.get
-
-Gets the active session, or returns `{status:inactive}` on error or no active session.
-
-- `callback` - Function that is called with the session once the request has completed.
-
-```javascript
-signIn.session.get(function (res) {
-  // Session exists, show logged in state.
-  if (res.status === 'ACTIVE') {
-    // showApp()
-  }
-  // No session, or error retrieving the session. Render the Sign-In Widget.
-  else if (res.status === 'INACTIVE') {
-    signIn.renderEl({
-        el: '#osw-container'
-      },
-      function success(res) {
-        // showApp() if res.status === 'SUCCESS'
-      },
-      function error(err) {
-        // handleError(err)
-      }
-    );
-  }
-});
-```
-
-### session.refresh
-
-Refresh the current session by extending its lifetime. This can be used as a keep-alive operation.
-
-- `callback` - Function that is called after the refresh request has completed.
-
-```javascript
-signIn.session.refresh(function (res) {
-  if (res.status === 'ACTIVE') {
-    // The session now has an extended lifetime
-  }
-  else if (res.status === 'INACTIVE') {
-    // There is no current session, render the Sign-In Widget
-  }
-});
-```
-
-### session.close
-
-Signs the user out of their current Okta session.
-
-- `callback` - Function that is called once the session has been closed. If there is an error, it will be passed to the callback function.
+Before version 3.0, the Sign-in Widget exposed `session` and `tokenManager` objects, and methods like `tokenManager.get`. These methods have been moved into the [AuthJS](https://github.com/okta/okta-auth-js#api) base library and are documented there.
```

Comment:
Mentioning what changed in 3.0 is good 👍 I think we could be a little more accurate, and describe that the previous methods were proxy methods which have been removed in favor of direct access.  We haven't actually moved anything to AuthJS, the methods were already there (tell me if I'm wrong?)

Reply:
Correct!

Reply:
"moved into" was my suggested wording, but I like @robertjd's better 🙂